reana_client/cli/workflow.py

```diff
@@ -272,18 +277,32 @@ def workflows_list(  # noqa: C901
 
         # Sort by given column, making sure that `None` is at the bottom of the list.
         sort_column_id = 2
-        if sort_columm_name.lower() in headers[type]:
+        if sort_columm_name.lower() == "run_number":
```

Comment:
Minor suggestion related with my previous comment: it would be possible to keep `workflow["size"]` value as dict and define a special sorting function to always sort by `raw` for `size` column: `lambda x: int(x[sort_column_id]["raw"])`.
Later in the bottom instead of `content = raw_to_human_readable[content]` you could use
```
if headers[type][i] in workspace_size_header:
    content = value.get(human_readable_or_raw)
```